src/main/resources/fabric.mod.json

```diff
@@ -22,6 +22,6 @@
   ],
   "depends": {
     "fabricloader": "*",
-    "minecraft": "1.16.2"
+    "minecraft": ">=1.16.2"
```

Comment:
We cannot support 1.17 either, it could change the code and break the mod.

Reply:
IDK mod.json syntax for 'between two versions', do you/

Reply:
Honestly, most mods I see just seem to have the metadata specify _approximately_ the version numbers it supports, and then just cross fingers and hope it doesn't break. Lots of mods in practice are labeled for supporting 1.16.2 but don't bother to update because 1.16.3 doesn't break. :)

Might be a good idea to simply specify version number on the .jar and then just release an update whenever it breaks, for the new version. Not like the feature set is going to expand with this mod... so I doubt you need to backport things ever.

Reply:
I found out we can use tilde range: `~1.16.2`

I prefer doing it this way because this being a mixing based mod makes it almost mandatory to break when 1.17 arrives.

Reply:
Ah, interesting! What's the syntax for that? I'll add it to the PR.